internal/lvgen/generate.go

```diff
@@ -552,22 +560,35 @@ func AddEnumVal(name, val string) error {
 	if err != nil {
 		return fmt.Errorf("invalid enum value %v = %v", name, val)
 	}
-	return addEnumVal(name, ev)
+	return addEnumVal(name, ev, nil)
+}
+
+// AddEnumValMeta will add a new enum value with attached metadata to the list.
```

Comment:
I'd like to see a comment here or somewhere else in this file describing what we mean by metadata, with an example from the libvirt sources. Just the comment block from one of the remote procedure enum values would do. Metadata is such a generic term otherwise.